### PR TITLE
V22 subtyping - chr instab (2/N)

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Run chromosomal instability

#### What was your approach?
This module also needed to be divided into script to run for subtyping and plot scripts to not run. I added that code to the bash script.

`01-localization-of-breakpoints.Rmd` was not set up to intake either histologies file, so I added that. However, when I tried to run the `OPENPBTA_BASE_SUBTYPING=1 bash ../analyses/chromosomal-instability/run_breakpoint_analysis.sh`, I get an error: 
```bash
Quitting from lines 150-154 (01-localization-of-breakpoints.Rmd) 
Error: '../../data/pbta-histologies.tsv' does not exist in current working directory ('/home/rstudio/kitematic/analyses/chromosomal-instability').
```

So I am not sure where the logic is going wrong.

In the meantime, I changed the param to 1 inside of the Rmd and ran it, to get the analysis file, in order to continue the PRs.

#### What GitHub issue does your pull request address?
#1426 partial

#### Which areas should receive a particularly close look?
logic for choosing histology file

#### Is there anything that you want to discuss further?
Also note, this 01 script requires short_histology

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/chromosomal-instability/01-localization-of-breakpoints.Rmd [hunk 1]
```diff
@@ -75,12 +79,19 @@ Here's all the input files we will need:
...
 chr_file <- file.path(data_dir, "WGS.hg38.strelka2.unpadded.bed")
-metadata_file <- file.path(data_dir, "pbta-histologies.tsv")
 breaks_file <- file.path(output_dir, "breaks_lists.RDS")
 uncallable_file <- file.path(
   "..", "copy_number_consensus_call", "ref",
   "cnv_excluded_regions.bed"
 )
+
+# set up histologies file
+if ( params$base_run == 0 ){
```
Comment (jaclyn-taroni) [inline analyses/chromosomal-instability/01-localization-of-breakpoints.Rmd:89]: Can you confirm that this runs as expected if you pass `0` as the `param`? I [can think of other cases](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e99c13550956c95fa83e9c811515e04a01a8c3d4/analyses/gene-set-enrichment-analysis/02-model-gsea.Rmd#L45) where `params` did not exactly behave as expected.

Comment (jharenza) [reply analyses/chromosomal-instability/01-localization-of-breakpoints.Rmd:89]: Do you mean within a v21 branch or within v22 branch which should error out?

Comment (jharenza) [reply analyses/chromosomal-instability/01-localization-of-breakpoints.Rmd:89]: while in this branch (v22 data download), running: 
```bash
root@334c2bfa1312:/home/rstudio/kitematic/scripts# OPENPBTA_BASE_SUBTYPING=0 bash ../analyses/chromosomal-instability/run_breakpoint_analysis.sh

Error: 
 Could not find needed file(s):
../../data/pbta-histologies.tsv
Check your options and set up.
Execution halted
```

This is expected. I will open a new PR with a v21 run from master.

Comment (jharenza) [reply analyses/chromosomal-instability/01-localization-of-breakpoints.Rmd:89]: Seems the param works if 1 is default, but not 0.

Comment (jaclyn-taroni) [reply analyses/chromosomal-instability/01-localization-of-breakpoints.Rmd:89]: I meant did you do functional testing by passing `params = list(base_run = 0)`. I just did locally.

**Review by jaclyn-taroni — APPROVED** [review]
👍🏻  tested the logic locally

**Pull request merged**